optimize/upgrade/src/test/java/io/camunda/optimize/upgrade/service/UpgradeValidationServiceIT.java

```diff
@@ -7,21 +7,19 @@
  */
 package io.camunda.optimize.upgrade.service;
 
-import static io.camunda.optimize.AbstractIT.OPENSEARCH_PASSING;
 import static io.camunda.optimize.upgrade.EnvironmentConfigUtil.createEmptyEnvConfig;
 import static io.camunda.optimize.upgrade.EnvironmentConfigUtil.deleteEnvConfig;
 import static org.assertj.core.api.Assertions.assertThatNoException;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 import io.camunda.optimize.upgrade.exception.UpgradeRuntimeException;
 import java.util.stream.Stream;
-import org.junit.jupiter.api.Tag;
+import org.camunda.optimize.upgrade.exception.UpgradeRuntimeException;
```

Comment:
This causes a compilation error, as org.camunda doesnt exist in mono-repo. Why was this added to begin with?